pkg/cmd/server/kubernetes/master_config.go

```diff
@@ -295,6 +286,22 @@ func BuildKubernetesMasterConfig(options configapi.MasterConfig, requestContextM
 	return kmaster, nil
 }
 
+// GetPluginConfigFile translates from the master plugin config to a file name containing
```

Comment:
Please move to a common package like pkg/cmd/server/admission and call from https://github.com/openshift/origin/blob/master/pkg%2Fcmd%2Fserver%2Forigin%2Fmaster_config.go#L183 as well.